cmd/convox/scale.go

```diff
@@ -35,6 +39,7 @@ func cmdScale(c *cli.Context) error {
 	// initialize to invalid values that indicate no change
 	count := -2 // -1 is valid, indicates removing the process and ELB
 	memory := -1
+	cpu := -1
```

Comment:
this seems relatively easy to do. I was imagining having a central mapping between count/cpu/mem "noop" vals somewhere and then referencing that here and in `SetFormation`. I would have made this change myself, but not sure how it would look in practice (new to go)